mostlyai/qa/_accuracy.py

```diff
@@ -1104,7 +1145,10 @@ def bin_non_categorical(
         )
         labels = [str(b) for b in breaks[:-1]]
 
-    new_labels_map = {label: f"⪰ {label}" for label in labels}
+    if label_style == ">= X":
+        new_labels_map = {label: f"⪰ {label}" for label in labels}
+    else:  # label_style == ">= X < Y"
+        new_labels_map = {label: f"⪰ {label} ≺ {next_label}" for label, next_label in zip(labels, labels[1:] + ["∞"])}
```

Comment:
Maybe worth noting the meaning of each of the styles in words.
E.g. which boundaries are displayed for each case.

Reply:
Introduced now `bounded` (`>= X < Y`) and `unbounded` (`>= X`) label styles.